style.css

```diff
@@ -0,0 +1,85 @@
+body {
+  /* position: relative; */
+  background: #fff;
+  margin: auto;
+  border: auto;
+}
+
+.bg {
+  background-image: url("./Images/background/bg.svg");
+  height: 648px;
+  background-repeat: no-repeat;
+}
```

Comment:
- [Optional] Kindly consider using flexbox to layout this section. You can use `display:flex;` `flex-direction: column;`.